### PR TITLE
Add com.ktechpit.orion

### PR DESCRIPTION
**Pull request by keshavbhatt**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by keshavbhatt** [comment]
bot, build com.ktechpit.orion

File: com.ktechpit.orion.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  - --filesystem=xdg-videos:rw
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --talk-name=org.freedesktop.FileManager1
+  - --own-name=org.mpris.MediaPlayer2.mpv.*
+  - --talk-name=org.gnome.SessionManager
```
Comment (hfiguiere) [inline com.ktechpit.orion.yaml]:
```suggestion
  - --device=all
  - --env=INSTALL_TYPE=flatpak
  - --filesystem=/tmp
  - --filesystem=xdg-download
  - --filesystem=xdg-pictures
  - --filesystem=xdg-run/gvfsd
  - --filesystem=xdg-videos
  - --share=ipc
  - --share=network
  - --socket=fallback-x11
  - --socket=pulseaudio
  - --socket=wayland
  - --talk-name=org.freedesktop.FileManager1
  - --own-name=org.mpris.MediaPlayer2.mpv.*
  - --talk-name=org.gnome.SessionManager
```


It's still a very broad set of permissions

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        - Torrent host server(create and share files by hosting a torrent)
+        - Torrent search engine plugin
+        - Discover movie browser plugin
+        - YouTube player & downloader plugin
+        - Torrent meta plugin (shows quick overview of selected torrent)
+        - History manager plugin (keeps your opened torrents safe for future)
+
+<br/>
+
+## Install Orion on Linux Desktop
```
Comment (hfiguiere) [inline README.md:40]: this README will NEVER be  seen by users.

File: com.ktechpit.orion.yaml (hunk 1 above)
Comment (keshavbhatt) [reply com.ktechpit.orion.yaml]: done, Thanks

File: README.md (hunk 2 above)
Comment (keshavbhatt) [reply README.md:40]: Is it okay to keep? :confused:

**Comment by keshavbhatt** [comment]
bot, build com.ktechpit.orion

**Comment by keshavbhatt** [comment]
Working as expected.

**Comment by keshavbhatt** [comment]
I see, I was wondering about the process. I thought maybe you guys have some fixed interval/or request threshold where you bulk publish all approved requests. I can only guess, maybe this is documented somewhere for transparency.

**Comment by keshavbhatt** [comment]
Its been two weeks, is there any reason to block this request for so long?

**Pull request closed without merging**